client/src/components/LibraryFolder/TopToolbar/import-to-history/import-collection.js

```diff
@@ -34,8 +34,9 @@ var ImportCollectionModal = Backbone.View.extend({
         this.histories = new mod_library_model.GalaxyHistories();
         return this.histories.fetch();
     },
-    createNewHistory: function (new_history_name) {
-        return axios.post(`${getAppRoot()}api/histories`, { name: new_history_name });
+    async createNewHistory(new_history_name) {
+        const { data } = await axios.post(`${getAppRoot()}api/histories`, { name: new_history_name });
```

Comment:
Thanks a lot @mvdbeek!!! I was wondering, how does he get that error.
it's a bit weird, that we create a new history and don't switch to it, I think we before
```suggestion
        const { data } = await axios.post(`${getAppRoot()}api/histories`, { name: new_history_name });
        getGalaxyInstance().currHistoryPanel.switchToHistory(data.id);
```

Reply:
Yeah, I think that's a good idea.